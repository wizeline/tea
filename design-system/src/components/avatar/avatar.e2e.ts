import { newE2EPage } from '@stencil/core/testing';

describe('wz-avatar', () => {
  it('should render an avatar-component', async () => {
    const page = await newE2EPage();
    await page.setContent(
      `<wz-avatar xlarge square src="https://api.adorable.io/avatars/285/abott@adorable.png" />`,
    );
    const avatar = await page.find('wz-avatar');
    expect(avatar.outerHTML).toMatchSnapshot();
  });
});
