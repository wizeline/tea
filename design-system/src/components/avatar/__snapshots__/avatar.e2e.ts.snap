// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wz-avatar should render an avatar-component 1`] = `"<wz-avatar xlarge=\\"\\" square=\\"\\" src=\\"https://api.adorable.io/avatars/285/abott@adorable.png\\" class=\\"hydrated\\"><div class=\\"square xlarge\\" style=\\"background-image: url(\\"https;\\"></div></wz-avatar>"`;
